docs/release.md

```diff
@@ -0,0 +1,117 @@
+Welcome to wxWidgets, a free and open source cross-platform C++ framework for
+writing advanced GUI applications using native controls.
```

Comment:
The uploader script will eventually use this file as the template for the GitHub release descriptions. Maybe we can find a good way to automatically inject the SHA1 checksums later, but no big deal if not.